backend/src/controllers/decisionControllers.js

```diff
@@ -12,6 +12,20 @@ const getDecisions = (req, res) => {
     });
 };
 
+const getDecisionCards = (req, res) => {
```

Comment:
you're not getting decision cards, but users. and not one decision but all decisions : getDecisionUsers with an id in params ? getDecisionsWithUsers ?